Slider/Assets/Scripts/UI/Artifact/Screens/ArtifactScreenAnimator.cs

```diff
@@ -121,8 +121,8 @@ private IEnumerator SwitchScreens(int target)
         }
 
         yield return new WaitForSeconds(duration);
-        
-        OnScreenChanged(target, currentScreenIndex);
```

Comment:
revert so its `OnScreenChanged(target, currentScreenIndex);` unless theres a reason you wanted to do this

Reply:
nvm i remember why you mentioned this